hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/CloneSnapshotProcedure.java

```diff
@@ -71,6 +75,7 @@
   private TableDescriptor tableDescriptor;
   private SnapshotDescription snapshot;
   private boolean restoreAcl;
+  private String cloneSFT;
```

Comment:
I think we need to persist this in the procedure state data?

Reply:
Thanks for pointing it out! I'm adding this.

Reply:
Looking at the code, I think there is already a bug in CloneSnapshotProcedure. We do not persist restoreAcl...

Let me file another issue to fix it.